go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/stackrox/rox
 
-go 1.22
-
-toolchain go1.22.5
+go 1.22.5
```

Comment:
how come this was changed? Did a dependency require this?

Reply:
That's my understanding.

Reply:
it's due to https://github.com/operator-framework/helm-operator-plugins/pull/385, which I commented on now because this is a rather disruptive change